master/usage/index.md

```diff
@@ -1,5 +1,67 @@
----
-title: Using Calico
----
-
+---
+title: Using Calico
+---
+
 This section contains information on using Calico.
+
+## Calico software components
```

Comment:
I want to think about what type of user is going to be coming to the "Usage" section - I imagine it's someone who has already deployed Calico after going through the "Getting Started" section, maybe has walked through a tutorial or two, and is now looking for next steps on how to perform common tasks.

I'm not sure this introduction captures that - we should think a bit more about the right wording.